src/gt4py/next/ffront/decorator.py

```diff
@@ -302,20 +245,41 @@ def __call__(self, *args, offset_provider: dict[str, Dimension], **kwargs) -> No
                 stacklevel=2,
             )
             with next_embedded.context.new_context(offset_provider=offset_provider) as ctx:
-                ctx.run(self.definition, *rewritten_args, **kwargs)
+                # TODO(ricoh): move into test
```

Comment:
I'd do this before this PR is merged. And then even split it into a separate PR.